config/gcpmanager/gcpmanager-skaffold.yaml.tpl

```diff
@@ -0,0 +1,31 @@
+apiVersion: skaffold/v3
+kind: Config
+metadata:
+  name: gcpmanager
+  labels: {}
+build:
+  tagPolicy:
+    gitCommit: {}
+  platforms: ["linux/amd64", "darwin/arm64"]
+  cluster:
+    resources:
+      requests:
+        cpu: 300m
+        memory: 512Mi
+    serviceAccount: container-builder
+  artifacts:
+    - image: us-central1-docker.pkg.dev/${PROJECT_ID}/substratus/gcpmanager
```

Comment:
I've found no better way to use these than to template them out, run them through `envsubst`, use the rendered file, and ignore that rendered file.